compiler/dyno/tools/chpldoc/chpldoc.cpp

```diff
@@ -645,7 +806,9 @@ struct RstSignatureVisitor {
 
     // storage kind
     if (f->thisFormal() != nullptr
-        && f->thisFormal()->storageKind() != IntentList::DEFAULT_INTENT) {
+        && f->thisFormal()->storageKind() != IntentList::DEFAULT_INTENT
+        && f->thisFormal()->storageKind() != IntentList::CONST_INTENT
+        && f->thisFormal()->storageKind() != IntentList::CONST_REF) {
```

Comment:
I would think we would want to print `const` or `const ref` in the documentation? Something to look at in follow-up PR.

Reply:
I think it's only like this because it was breaking `chpldoc` tests, but it may be one of those cases where the new behavior is preferred.